### DISCUSSION TITLE
Ollama GUI Mac Application Wrapper

### DISCUSSION THREAD

**Discussion by worldoptimizer** [open-webui/open-webui#257] [Ideas]
Is there any ideas or a chance that this would be possible? A electron (or something more modern) wrapper for this so we can run it easily on Mac?

**Comment by justinh-rahb** [comment]
Not sure I see the point? There are already other desktop apps that use the Ollama API if that's your style. You could just as easily make this webapp a PWA, then there's no "installation" required and it will work on any platform.

**Reply by worldoptimizer** [reply]
Any idea how… it doesn't feel that easy as I need to run docker etc.

**Reply by justinh-rahb** [reply]
"No installation for the user", I should have clarified. The server still needs to be setup. The author has made it quite clear that Docker is their only supported method of installation right now, for the sake of simplicity and keeping people's experience consistent.

**Comment by adan89lion** [comment]
I'm on macOS Sonoma, and I use Safari's new "Add to Dock" feature to create an applet on Dock (and in Launchpad) to run in a separate window. Additionally, launching the app doesn't require to run Safari, as it will launch as a new instance. The memory consumption is even better than the generic Electron apps.

**Comment by jfbloom22** [comment]
Like @adan89lion, I am enjoying the "Add to Dock" feature in Sonoma for WebUI.  One feature I am missing is the ability to open a mini chat like I do with ChatGPT: [Option Space].

**Reply by m8dhouse** [reply]
https://github.com/open-webui/open-webui/discussions/8099
We created a very basic Mac app for this. If there is sufficient interest I can clean up and make available.
